scenarios/kubernetes_e2e.py

```diff
@@ -549,7 +549,7 @@ def set_up_aws(workspace, args, mode, cluster, runner_args):
     mode.add_aws_runner()
 
 def read_gcs_path(gcs_path):
-    """reads a gcs path (gs://...) by GETing storage.googleapis.com"""
```

Comment:
this is intentionally because it does an HTTP `GET` request

Reply:
It probably should say `GET-ing` or something slightly more verbose like `by HTTP GET to storage.googleapis.com`

Reply:
thanks a lot! Already updated it.